### ISSUE TITLE
Exported VST instance (not restricted) won't show Property Pane Tabs.

### ISSUE THREAD

**Issue by damiensellier** [damiensellier/CtrlrX#40]
When exporting a non-restricted VST & VST3 instance, the exported VST won't show the Property Pane Tabs on the right side.

<img width="603" alt="Capture d’écran 2024-12-09 à 00 27 44" src="https://github.com/user-attachments/assets/5900f82e-c119-4893-8044-fdd287f2aa9e">

**Comment by damiensellier** [comment]
LOOK FOR : CtrlrPanelComponentProperties

ALSO : CtrlrEditor.cpp is were the layout is defined depending on if Ctrlr is a plugin or standalone, restricted or not, with menuBar or not etc

**Comment by damiensellier** [comment]
Check if tabs are showing up if set at left position

addAndMakeVisible (tabbedComponent = new TabbedComponent (TabbedButtonBar::TabsAtRight));

**Comment by damiensellier** [comment]
CtrlrManagerInstance.cpp


```
void CtrlrManager::setEmbeddedDefaults()
{
	_DBG("CtrlrManager::setEmbeddedDefaults");

	/* first init the ApplicationProperties stuff */
	if (ctrlrPlayerInstanceTree.hasProperty(Ids::restricted))
	{
		ctrlrPlayerInstanceMode	= (CtrlrInstance)(int)ctrlrPlayerInstanceTree.getProperty(Ids::restricted);
	}
	else
	{
		ctrlrPlayerInstanceMode = InstanceSingle;
	}

	ctrlrProperties.reset(new CtrlrProperties (*this));

	setProperty (Ids::ctrlrLogToFile, false);
	setProperty (Ids::ctrlrLuaDebug, false);
	setProperty (Ids::ctrlrVersionSeparator, "_");
	setProperty (Ids::ctrlrVersionCompressed, false);
	setProperty (Ids::ctrlrMidiMonInputBufferSize, 8192);
	setProperty (Ids::ctrlrMidiMonOutputBufferSize, 8192);
	setProperty (Ids::ctrlrLuaDisabled, false);
	setProperty (Ids::ctrlrOverwriteResources, true);
	setProperty (Ids::ctrlrAutoSave, true);
	setProperty (Ids::ctrlrAutoSaveInterval, 300);
	setProperty (Ids::ctrlrLogOptions, 32);
	setProperty (Ids::ctrlrUseEditorWrapper, (bool)ctrlrPlayerInstanceTree.hasProperty(Ids::ctrlrUseEditorWrapper) ? (bool)ctrlrPlayerInstanceTree.getProperty(Ids::ctrlrUseEditorWrapper) : true);
	setProperty (Ids::uiLuaConsoleInputRemoveAfterRun, true);
}
```

**Comment by damiensellier** [comment]
FIXED : v5.6.31

CtrlrManagerInstance.cpp


```
void CtrlrManager::setEmbeddedDefaults()
{
	_DBG("CtrlrManager::setEmbeddedDefaults");

	/* first init the ApplicationProperties stuff */
	if (ctrlrPlayerInstanceTree.hasProperty(Ids::restricted))
	{
		ctrlrPlayerInstanceMode	= (CtrlrInstance)(int)ctrlrPlayerInstanceTree.getProperty(Ids::restricted);
	}
	else
	{
		ctrlrPlayerInstanceMode = InstanceSingle;
	}

	ctrlrProperties.reset(new CtrlrProperties (*this));

	setProperty (Ids::ctrlrLogToFile, false);
	setProperty (Ids::ctrlrLuaDebug, false);
	setProperty (Ids::ctrlrVersionSeparator, "_");
	setProperty (Ids::ctrlrVersionCompressed, false);
	setProperty (Ids::ctrlrMidiMonInputBufferSize, 8192);
	setProperty (Ids::ctrlrMidiMonOutputBufferSize, 8192);
	setProperty (Ids::ctrlrLuaDisabled, false);
	setProperty (Ids::ctrlrOverwriteResources, true);
	setProperty (Ids::ctrlrAutoSave, true);
	setProperty (Ids::ctrlrAutoSaveInterval, 300);
    // setProperty (Ids::ctrlrLogOptions, 32);
    setProperty (Ids::ctrlrLogOptions, 6014); // Added v5.6.31. 6014 shows everything by default with MIDI messages in Hex
    setProperty (Ids::ctrlrScrollbarThickness, 16.0f); // Added v5.6.31
    setProperty (Ids::ctrlrMenuBarHeight, 24); // Added v5.6.31
    setProperty (Ids::ctrlrTabBarDepth, 24); // Added v5.6.31
	setProperty (Ids::ctrlrUseEditorWrapper, (bool)ctrlrPlayerInstanceTree.hasProperty(Ids::ctrlrUseEditorWrapper) ? (bool)ctrlrPlayerInstanceTree.getProperty(Ids::ctrlrUseEditorWrapper) : true);
	setProperty (Ids::uiLuaConsoleInputRemoveAfterRun, true);
}
```
